addons/mail/static/src/models/message/message.js

```diff
@@ -465,6 +465,37 @@ function factory(dependencies) {
                 this.originThread.isModeratedByCurrentPartner
             );
         }
+        /**
+         * @private
+         * @returns {boolean}
+         */
+        _computeIsSubjectSimilarToOriginThreadName() {
+            if (
+                !this.subject ||
+                !this.originThread ||
+                (this.originThread && !this.originThread.name)
```

Comment:
no need to check for `this.originThread` on this line, the OR will stop at the previous line if it's not set